code/__DEFINES/misc.dm

```diff
@@ -492,6 +492,7 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 
 // Play time / EXP
 #define PLAYTIME_HARDCORE_RANDOM 120
+#define PLAYTIME_VETERAN 10000 HOURS
```

Comment:
this is in minutes so this is 6000000 hours

Reply:
ur 6 billion hours or whatever

Reply:
ok so? just play more casual